EIPS/eip-2537.md

```diff
@@ -291,11 +291,11 @@ The cost of the pairing check operation is `43000*k + 65000` where `k` is a numb
 
 #### Fp-to-G1 mapping operation
 
-Fp -> G1 mapping is `5500` gas.
+Fp -> G1 mapping is `4125` gas.
 
 #### Fp2-to-G2 mapping operation
 
-Fp2 -> G2 mapping is `75000` gas
+Fp2 -> G2 mapping is `14055` gas
```

Comment:
Maybe let's put 15000 here as you have in your NUC benchmarks. This number of from my BLST benchmarks. Considering this was 5x higher originally and nobody complained...

Reply:
Updated to 17000 based on the feedback from @jwasinger